src/vm/prestub.cpp

```diff
@@ -224,13 +224,17 @@ void DACNotifyCompilationFinished(MethodDesc *methodDesc)
 
         _ASSERTE(modulePtr);
 
+#ifndef FEATURE_GDBJIT
```

Comment:
I don't understand why GDBJIT always needs this SOS notification or why would it need the notification at all?